app/views/individual/usermatching/CheckYourUserDetails.scala.html

```diff
@@ -86,8 +86,12 @@
   summaryRow(rowName, rowQuestion, rowAnswer, rowUrl, hiddenContent)
 }
 
-@principalMainTemplate(title = messages("summary.title"), backLink = Some(backUrl)) {
-  <h1 class="govuk-heading-l">@Messages("user-details.summary.heading")</h1>
+@principalMainTemplate(title = messages("user-details.summary.title"), backLink = Some(backUrl)) {
+
+<header class="hmrc-page-heading">
+    <h1 class="govuk-heading-l">@Messages("user-details.summary.heading")</h1>
+    <p class="hmrc-caption govuk-caption-xl"><span class="govuk-visually-hidden">@Messages("base.caption")</span>@Messages("user-details.summary.caption")</p>
+</header>
```

Comment:
The caption needs to be placed before the heading:
```suggestion
<header class="hmrc-page-heading">
    <p class="hmrc-caption govuk-caption-xl"><span class="govuk-visually-hidden">@Messages("base.caption")</span>@Messages("user-details.summary.caption")</p>
    <h1 class="govuk-heading-l">@Messages("user-details.summary.heading")</h1>
</header>
```